Xamarin.Forms.Core.UnitTests/BindingBaseUnitTests.cs

```diff
@@ -166,17 +166,25 @@ public void ChangeAfterApply()
 		[Test]
 		public void StringFormatNonStringType()
 		{
-			var property = BindableProperty.Create("Foo", typeof(string), typeof(MockBindable));
-			var binding = new Binding("Value", stringFormat: "{0:P2}");
+			var currentCulture = System.Threading.Thread.CurrentThread.CurrentCulture;
+			try
+			{
+				System.Threading.Thread.CurrentThread.CurrentCulture
+					= System.Globalization.CultureInfo.InvariantCulture;
 
-			var vm = new  { Value = 0.95d };
-			var bo = new MockBindable { BindingContext = vm };
-			bo.SetBinding(property, binding);
+				var property = BindableProperty.Create("Foo", typeof(string), typeof(MockBindable));
+				var binding = new Binding("Value", stringFormat: "{0:P2}");
+
+				var vm = new { Value = 0.95d };
+				var bo = new MockBindable { BindingContext = vm };
+				bo.SetBinding(property, binding);
 
-			if (System.Threading.Thread.CurrentThread.CurrentCulture.Name == "tr-TR")
-				Assert.That(bo.GetValue(property), Is.EqualTo("%95,00"));
```

Comment:
that part of the test is now missing

Reply:
Hi , thanks for your review.
I had understood that the test StringFormatNonStringType wold to check if StringFormat behavior is correct with non-string values indifferently of the culture.

What do you think if I use [TestCaseSource](https://github.com/nunit/docs/wiki/TestCaseData) to test the behavior for each culture?